kimchi/src/circuits/lookup/runtime_tables.rs

```diff
@@ -10,16 +10,65 @@ use ark_ff::Field;
 use serde::{Deserialize, Serialize};
 
 /// The configuration of a runtime table.
+/// Note that its table ID these should be chosen not to collide with other runtime tables' IDs,
+/// as well as other types of lookup tables and their IDs.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct RuntimeTableConfiguration {
-    /// The table id.
-    /// Note that these should be chosen not to collide with other runtime tables' IDs,
-    /// as well as other types of lookup tables and their IDs.
+    /// The table ID.
     pub id: i32,
-    /// The length of the runtime table.
+    /// The number of entries contained in the runtime table.
     pub len: usize,
 }
 
+/// At setup time, a runtime table can be configured to be an indexed table,
+/// or a custom table.
+/// An index table has a counter starting from 0 in its first column,
+/// whereas a custom table has a given list of field elements.
+pub enum RuntimeTableSetup<F> {
+    /// An indexed runtime table has a counter, starting from zero, as first column.
+    Indexed(RuntimeTableConfiguration),
+    /// Unliked an indexed runtime table, the first column is fixed at setup time.
```

Comment:
Nit: `Unlike`

Reply:
Nit: both are fixed at setup time, it's just that the first column isn't necessarily sequential here.